check.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# LICENSE: Public Domain
+
+
+echo "CHECK that All translator have a translatorID"
+haveid=$(grep -c '"translatorID"' *.js | awk -F: '$2!="1" {print $1,"found",$2,"translatorID"}')
+if [[ -n "$haveid" ]];then
+  echo "Error: no translatorID or multiple translatorIDs"
+  echo "$haveid"
+  exit 1
+fi
+
+echo "CHECK that all translatorIDs are unique"
+noduplicate=$( { (grep '"translatorID"' *.js | awk -F: '{print $3}') & (awk -F# 'NR>2 { print $1 }' deleted.txt) } | sed 's/[", ]//g' | sort | uniq -cd)
```

Comment:
Oh, well, sort of — that's just the usual background operator (e.g., `firefox &`), but in this case it's causing the first command to run in the background before being concatenated into the parent shell. So, e.g., `foo=$({ (sleep 2; echo a) & (echo b) }); echo $foo` results in "b a" instead of "a b" for a semicolon.

This isn't a pattern I've ever seen used before in shell scripts, and for instantaneous commands like this it seems unnecessarily confusing (and could possibly make debugging confusing if a race condition affects the order of the output). Better to just use a `;`.